src/Makefile

```diff
@@ -51,6 +51,7 @@ endif
 ifeq ($(OS),Windows_NT)
 	PLATFORM = Windows
 else
+	CFLAGS += -O3
```

Comment:
I do not know why compiling with -O3 makes tests fail on Windows.

Reply:
Nim `-d:release` uses `-O3` by default.